docs/reference/commandline/build.md

```diff
@@ -454,20 +454,27 @@ Specifying the `--isolation` flag without a value is the same as setting `--isol
 
 ### <a name="add-host"></a> Add entries to container hosts file (--add-host)
 
-You can add other hosts into a container's `/etc/hosts` file by using one or
-more `--add-host` flags. This example adds a static address for a host named
-`docker`:
+You can add other hosts into a build container's `/etc/hosts` file by using one
+or more `--add-host` flags. This example adds static addresses for hosts named
+`docker` and `dockerv6`:
 
 ```console
-$ docker build --add-host docker:10.180.0.1 .
+$ docker build --add-host docker=10.180.0.1 --add-host dockerv6=2001:db8::33 .
 ```
 
 If you need your build to connect to services running on the host, you can use
 the special `host-gateway` value for `--add-host`. In the following example,
 build containers resolve `host.docker.internal` to the host's gateway IP.
 
 ```console
-$ docker build --add-host host.docker.internal:host-gateway .
+$ docker build --add-host host.docker.internal=host-gateway .
+```
+
+IPv6 addresses may be enclosed in square brackets, and a `:` separator is
+accepted. For example:
```

Comment:
```suggestion
You can wrap an IPv6 address in square brackets.
`=` and `:` are both valid separators.
Both formats in the following example are valid:
```

Reply:
That's better! Thank you ... and ditto all the rest, I've pushed the changes.